pnpm-lock.yaml

```diff
@@ -1,9 +1,5 @@
 lockfileVersion: '6.0'
 
-settings:
```

Comment:
Do you know why this file changed?

Reply:
I tried to add sass/scss support at one point, and then changed my mind to not overcomplicate this change. Re-running pnpm install again after removing the sass stuff caused it to bump this file